game/src/graphic/textures/TextureHandler.java

```diff
@@ -38,6 +68,12 @@ private void addAllAssets(FileHandle fh) {
         }
     }
 
+    /**
+     * Returns all available asset paths, that was found. Can be used with {@link
+     * TextureHandler#getTexturePaths(String)}.
+     *
+     * @return Returns all available asset paths, that was found.
+     */
     public Set<String> getAvailablePaths() {
```

Comment:
muss das wirklich in die api? ist das wichtig nach außen (für den user) oder ist das eine hilfsmethode?

Reply:
Mir fällt spontan kein use case dafür ein, weshalb die Studis diese Methode brauchen sollten. Die Studis wissen ja, bzw. schauen vorher nach, welche Texturen sie verwenden wollen, und sie suchen sich diese ja nicht "programmatisch" zur Laufzeit heraus ... Außerdem/Bzw. ist das doch eigentlich zum Teil die Aufgabe des Handlers

Vielleicht hat noch jemand anderes eine Idee (@AMatutat ?), aber sonst würde ich die Methode private machen

Reply:
wenn du als derjenige, der diese methode beisteuert, keinen use-case für "public" hast, dann ist es "privat". da brauchst du niemanden noch fragen.

Reply:
So. Der Kommentar sagt, dass diese Methode nur für die Unit-Tests da ist. Damit gehört sie nicht in die API oder sogar ganz raus.